### PR TITLE
Setup of fund details

### PR DESCRIPTION
**Pull request by line47**
First pass at setting up the fund page. 🚀 
**Original PDF**
[tsp.gov-S_Fund_Detail_Page.pdf](https://github.com/frtib/tsp-redesign/files/2263500/tsp.gov-S_Fund_Detail_Page.pdf)

**Screenshot**
funds

### CODE REVIEW

File: pages/fund-basics/s-fund.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    answer: There is a risk of loss if the Dow Jones U.S. Completion TSM Index declines in response to changes in overall economic conditions (market risk) or if the S Fund does not grow enough to offset the reduction in purchasing power (inflation risk).
+  - question: How can I use the S Fund in my TSP?
+    answer: The S Fund can be useful in a portfolio that also contains stock funds that track other indexes. The C, S, and I Funds, for example, track different segments of the overall stock market without overlapping. By investing in all segments of the stock market (as opposed to just one), you reduce your exposure to market risk.
+avg_annual_returns:
+    ytd: +12.89%
+    one_yr: +13.89%
+    three_yr: +14.89%
+    five_yr: +15.89%
+    ten_yr: +16.89%
+---
```
Comment (line47) [inline pages/fund-basics/s-fund.md]: Added a bunch of front matter settings for this layout ☝️ @chachi69

**Comment by line47** [comment]
@chachi69 @Lapedra I was also wondering what the reasoning was behind changing the style of the accordions for this page and if it would be a good idea to style them consistently across the site?

**Comment by Lapedra** [comment]
@line47 

Green used on the annual percentages #2e8540
Yellow/orange used for active tab #FDB81E
Blue used for the background on the investment objective callout #046B99
Grey of the horizontal line for the header #B5B3B3
Grey of the horizontal lines for the annual percentages #F1F1F1

**Comment by Lapedra** [comment]
@line47 I didn't want to use the accordion style for the questions. I like the Plus sign for the questions/info that we're adding at the top of some pages. Will this be a problem?

**Comment by Lapedra** [comment]
Also, for your reference, I'm using the colors from the USWD.

**Comment by line47** [comment]
@Lapedra not a problem at all, I just wanted to ask before I work on the custom styles 👍 
Thanks for all this info!

**Comment by Lapedra** [comment]
Thank you. Can we use the [plus-circle] icon in Feather?

**Comment by line47** [comment]
For all accordions? or just on this page?

**Comment by line47** [comment]
@Lapedra one other color I forgot to ask about was the color of the text used for inactive tabs at the bottom?

**Comment by Lapedra** [comment]
Just on this page. 

Sidebar: I'm trying to be consistent, so some items may change. So, thank you for your patience. :)

**Comment by chachi69** [comment]
@line47 Since this is a first pass, do you want me to approve and deploy now or do you just want feedback?

**Comment by line47** [comment]
@chachi69 I'm open to whatever would work best for you all 👍

**Comment by chachi69** [comment]
@line47 I'm going with approve and deploy, because that's the only way Lapedra can see the working web page.

@Lapedra we probably should get your Mac back online with Atom and GitHub. That way you'll be able to pull a working branch down to your machine and preview it locally using `http://localhost:4000/`

**Pull request merged**